pynestml/codegeneration/nest_declarations_helper.py

```diff
@@ -95,3 +97,22 @@ def print_size_parameter(cls, ast_declaration):
         :rtype: str
         """
         return ast_declaration.get_size_parameter()
+
+    @classmethod
+    def print_delay_parameter(cls, variable: VariableSymbol, reference_converter) -> str:
```

Comment:
Can't this remain in the NEST printer class?

Reply:
Yes, it should go into a template. I can take a look at this and move it to a template later in a separate PR.